front/src/components/LanguageChanger/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+
+import InputWithList from '../UI/InputWithList';
+import classes from './LanguageChanger.module.css';
+
+export const LanguageChanger = ({ defaultLanguage }) => {
+  const { t, i18n } = useTranslation();
+  const changeLanguage = lng => {
+    i18n.changeLanguage(lng);
+  };
+
+  const langs = [
+    { name: 'English', id: 'en' },
+    { name: 'Русский', id: 'ru' },
+    { name: 'Česky', id: 'cz' },
+  ];
```

Comment:
What are the consequences of having this declaration here for component lifecycle and testing?